docs/pages/run/validator-management/proposer-config.md

```diff
@@ -0,0 +1,36 @@
+# Proposer Configuration
+
+:::warning
+This is an alpha feature. The feature and its format are subject to change.
+:::
+
+With Lodestar's validator client, you can assign specific metadata for each proposer/public key using a proposer configuration file written in YAML file. This will allow you to set specific graffiti, fee recipients and builder settings per validator key.
+
+### Example proposer_config.yaml
+
+```yaml
+proposer_config:
+  "0xa99a76ed7796f7be22d5b7e85deeb7c5677e88e511e0b337618f8c4eb61349b4bf2d153f649f7b53359fe8b94a38e44c":
+    graffiti: "graffiti"
+    strict_fee_recipient_check: false
+    fee_recipient: "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+    builder:
+      enabled: true
```

Comment:
for sake of completeness of the example could also add `selection` and `boost_factor`

see for all options
https://github.com/ChainSafe/lodestar/blob/fa969ca6fd12fe8c17a11c5536745a501284293e/packages/cli/src/util/proposerConfig.ts#L12